frontend_build/src/extract_$trs.js

```diff
@@ -67,18 +70,19 @@ extract$trs.prototype.apply = function(compiler) {
             }
           });
           if (messageNameSpace) {
-            // Every message needs to be namespaced - don't pollute our top level!
-            Object.keys(messages).forEach(function(key) {
-              // Create a new message id from the name space and the message id joined with '.'
-              var msgId = messageNameSpace + '.' + key;
-              if (messageExport[msgId]) {
-                // Warn about duplicate ids *within* a bundle (no way to warn across).
-                logging.error('Duplicate translation id ' + msgId + ' found in ' + module.resource)
-              } else {
-                // If all is good, save it onto our export object for the whole bundle.
+            // Warn about duplicate nameSpaces *within* a bundle (no way to warn across).
+            if (nameSpaces.includes(messageNameSpace)) {
```

Comment:
This is not a function. I think you are looking for:
`if (nameSpaces.indexOf(messageNameSpace) > -1) {`